alica_tests/autogenerated/include/BehaviourCreator.h

```diff
@@ -0,0 +1,23 @@
+#ifndef BEHAVIOURCREATOR_H_
```

Comment:
pragma once

Reply:
not modifying the autogenerated files for now.